File: template/Java/FizzBuzz.java
public class FizzBuzz {
	public static void main(String[] args) {
		for (int i = 1; i <= FIZZBUZZ_CYCLES; ++i) {
			boolean div3 = (i % 3 == 0);
			boolean div5 = (i % 5 == 0);
			if (div3 || div5) {
				System.out.println((div3 && div5) ? "FizzBuzz" : div3 ? "Fizz" : "Buzz");
			} else {
				System.out.println(i);
			}
		}
	}
}

File: template/C/fizz-buzz.c
#include <stdio.h>
#include <stdbool.h>

int main()
{
    char num_buf[10];
    for (int i = 1 ; i <= FIZZBUZZ_CYCLES; ++i)
    {
        const bool div3 = (i % 3 == 0);
        const bool div5 = (i % 5 == 0);
        printf("%s\n", ((div3 && div5) ? "FizzBuzz" :
            div3 ? "Fizz" :
            div5 ? "Buzz" :
            (sprintf(num_buf, "%d", i), num_buf)
        ));
    }

    return 0;
}
